### ISSUE TITLE
doki theme for normal visual studio

### ISSUE THREAD

**Issue by Skided** [doki-theme/doki-theme-vscode#43]
can you please add support for normal visual studio and not only for vsc?

**Comment by Unthrottled** [comment]
:thinking: I'm not opposed to the idea. I haven't had the need to use Visual Studio (not a .net dev), so that's why it doesn't exist. The answer is _eventually_. I've got some other things that I want to do before I can look at this though.

[Link for future me](https://docs.microsoft.com/en-us/visualstudio/extensibility/extensibility-hello-world?view=vs-2019)

**Comment by RichanHatsune** [comment]
👍  A Doki Theme for the full version of Visual Studio would be really nice. I already use the Doki Theme for my whole JetBrains Toolbox and VS Code. 

For Visual Studio I currently use a workaround with an selfmade custom Color Theme and the AddIn [ClaudiaIDE](https://github.com/buchizo/ClaudiaIDE) to add a Miku Hatsune wallpaper to my Code Editor:
![Screenshot 2021-05-20 114312](https://user-images.githubusercontent.com/42713468/118957185-97389980-b960-11eb-95be-92213a5f601d.png)

So it would be technically possible to create a Doki Theme Pack for Visual Studio. The only thing that will not be possible are stickers. Writing Visual Studio Extensions also takes a lot of work compared to VSCode.

**Comment by Unthrottled** [comment]
Noice! I'll have to take a peek later to see how much effort it would be make it work the way I want it to. Thanks for making this more probable of being a thing.

**Comment by Unthrottled** [comment]
![Screenshot 2021-07-20 061112](https://user-images.githubusercontent.com/15972415/126480481-cb84c7f1-dfe6-4b7c-a77d-7fae78e11190.png)


@RichanHatsune Thank you for pointing me in the right direction with ClaudiaIDE! Reading that codebase was an interesting experience 😅 .

You where right about the amount of effort it takes to build a theme/extension. There are over 2700 color configurations, hence why MVP doesn't have any light themes.

That being said I probably missed colors in places, so feel free to submit feedback to help make it better. I developed the VisualStudio extension using Rider, so my knowledge/usage of VisualStudio is (and will be) low. I did make the things I did see look pretty though.

[Link to the codebase with documentation on how to get started](https://github.com/doki-theme/doki-theme-visualstudio).
